tedana/workflows/tedana.py

```diff
@@ -11,6 +11,7 @@
 import argparse
 import numpy as np
 import pandas as pd
+from bs4 import BeautifulSoup, NavigableString
```

Comment:
I don't want to add a new dependency if at all possible... could we try using the `<object>` tag ?